src/Components/PromiseList/index.js

```diff
@@ -10,11 +10,14 @@ export type Props = {
 
 const PromiseList = (props: Props) => {
   const { promises } = props;
+  if (!promises) {
```

Comment:
The thought process here is that `promises = null` represents the API call hasn't been made.

So in the future this check would represent a spinner.

But you're correct that the empty array isn't being thought of.
Something to improve upon in the future